src/App.tsx

```diff
@@ -52,6 +52,14 @@ const App = () => {
                 return <routes.Edit {...props} {...{ videoId, segmentId }} />;
               }}
             />
+            <Route
+              path="/contact/:videoId?"
```

Comment:
This is a result of bad route structuring from early app development. It looks like there are still some issues when I visit the contact page.

I'm working to resolve these route issues in my current branch, so I will try to finish that branch soon and merge in the changes here before we merge this branch into production.

Reply:
ok. It still tries to get videos and shows an error.

Reply:
Is it done in the last PR?

Reply:
No this isn't done yet, sorry. I will focus on getting the PR with this fix finished 😄

Reply:
No problem. No hurry.